src/test/regress/multi_1_schedule

```diff
@@ -339,7 +340,6 @@ test: ssl_by_default
 # ---------
 test: distributed_types distributed_types_conflict disable_object_propagation distributed_types_xact_add_enum_value
 test: check_mx
-test: turn_mx_off
```

Comment:
I think we might need to replace `turn_mx_on` few lines below with `check_mx`, that'd help Ozan to find out which tests should be updated.

Reply:
As distributed_functions turns mx off within the tests, we can't. Hope explicit turn_mx_on can also help Ozan :)